src/App.js

```diff
@@ -120,6 +122,7 @@ render() {
       <SortResults
         searchStatus={this.state.isPerformingSearch}
         updateSortCriteria={this.updateSortCriteria}
+        disabled={ this.state.results === 1 }
```

Comment:
I think there may be a small bug here - what would the behaviour be if zero results were returned?

Reply:
If zero results return, an alert displays with Error Message: 'No Movies Found'. A Sort Results button does not render. Good point though that I could have covered this in my tests as an eventuality.